### PR TITLE
CI/ update eslint & codeQL workflows

### PR DESCRIPTION
**Pull request by Bootoyka**
### Security and eslint code scanning now able

This Pull Request gives you the opportunity to check security and good implementation of your code in the current project.
Now on every push / merge on main you can see if you pull request is eslint friendly

> Please check the eslint scan and try to fix you part @Cavonstavant @aurelenc @majent4 @EmileEttel

### CODE REVIEW

**Review by majent4 — APPROVED** [review]

**Review by Cavonstavant — APPROVED** [review]

**Comment by majent4** [comment]
Good report, made the changes on my end. I expected more warnings

**Review by EmileEttel — APPROVED** [review]
LGTM

File: .eslintrc.js [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-        "SharedArrayBuffer": "readonly"
-    },
-    "parserOptions": {
-        "ecmaVersion": 11,
-        "sourceType": "module"
-    },
-    "rules": {
-        "indent": [
-            "error",
-            2
```
Comment (aurelenc) [inline .eslintrc.js:19]: 4

**Review by aurelenc — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (aurelenc) [reply .eslintrc.js:19]: We can stay with 2 if you want, I don't care anyway but it's better to use 4 spaces to be along with the standards

Comment (Bootoyka) [reply .eslintrc.js:19]: as u wish

**Pull request merged**